lte/gateway/docker/mme/Dockerfile.ubuntu18.04

```diff
@@ -8,6 +8,7 @@ ARG FEATURES=mme_oai
 ENV MAGMA_ROOT=/magma
```

Comment:
Should we consider upgrading to Ubuntu 20.04?

Reply:
cc @rdefosse